pkg/ai/prompts.go

```diff
@@ -0,0 +1,8 @@
+package ai
```

Comment:
This is just a suggestion but shouldn't we rename this file to `common.go` ?

Reply:
I think `prompts.go` is more intuitive tbh if we see a need for more common functionality we rename then